src/panel/poi/blocks/Details.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
+import StarsBlock from './Stars';
 import AccessibilityBlock from './Accessibility';
-// import BreweryBlock from './Brewery';
 import InternetAccessBlock from './InternetAccess';
 import DeliveryBlock, { hasActiveDeliveryModes } from './Delivery';
 import { Divider } from 'src/components/ui';
 import { findBlock } from 'src/libs/pois';
 import { useI18n } from 'src/hooks';
+import poiSubClass from 'src/mapbox/poi_subclass';
 
 const DetailsBlock = ({ poi }) => {
   const { _ } = useI18n();
-
+  const subclass = poiSubClass(poi.subClassName);
   const accessibility = findBlock(poi.blocks, 'accessibility');
   const internetAccess = findBlock(poi.blocks, 'internet_access');
   const delivery = findBlock(poi.blocks, 'delivery');
-  // const brewery = findBlock(poi.blocks, 'brewery');
-  // {brewery && <BreweryBlock block={brewery} />}
+  // const stars = findBlock(poi.blocks, 'stars');
+  // Example:
+  const stars = {
```

Comment:
Just to be sure, this fake data is just a subpart of the stars block. It's ok for now to focus on the rendering, but there will still be the actual parsing and filtering to implement.
Maybe you can already use the "real" format:
```json
{
  "type": "stars",
  "ratings": [
    {
      "has_stars": "yes",
      "nb_stars": 4,
      "kind": "restaurant",
     } 
  ]        
}
```

Reply:
sure, though the last time I discussed this with Adrien today, "kind" actually seemed unnecessary. poi.subclass seemed enough. I'll leave it out for now but of course, if any use case appears, let's put it back